README.md

```diff
@@ -22,6 +22,8 @@ TwitterRestClient Examples
 TwitterStreamingClient Examples
 --------------------------------
 - [UserMentionStream](https://github.com/DanielaSfregola/twitter4s-demo/blob/master/src/main/scala/streaming/UserMentionStream.scala) receive stream of tweets mentioning yourself.
+- [TweetTimelineStreamPrinter](https://github.com/DanielaSfregola/twitter4s-demo/blob/master/src/main/scala/streaming/TweetTimelineStreamPrinter.scala) print the tweets in your timeline using stream.
+- [StreamMentionReplyier](https://github.com/DanielaSfregola/twitter4s-demo/blob/master/src/main/scala/streaming/StreamMentionReplyier.scala) mark as favorite all tweets with hashtag "scala".
```

Comment:
I am a bit concern about people running this project to try things out and suddenly generating lots and lots of activity in their profile. Why don't we just substitute the action of marking a tweet as favorite with a simple println? Something like "this tweet is cool! You should definitively like it: \n\t ${tweet.text}" 

...better safe than having people complaining that we messed with their account ;)